delta/service/src/test/resources/permissions-test.conf

```diff
@@ -0,0 +1,14 @@
+include "akka-test.conf"
+
+akka {
+  actor {
+    serialization-bindings {
```

Comment:
This is duplicated, already present in `akka-test.conf`

Reply:
it needs to be in this file, I'll remove it from the other